flopy/modflow/mfdis.py

```diff
@@ -939,11 +939,13 @@ def load(f, model, ext_unit_dict=None, check=True):
             print('   loading delr...')
         delr = Util2d.load(f, model, (ncol,), np.float32, 'delr',
                            ext_unit_dict)
+        delr = delr.array.reshape(ncol)

```

Comment:
Not sure why delr is being converted to a numpy array instead of leaving as a Util2d object.